### ISSUE TITLE
Stachtrace in pool's collector

### ISSUE THREAD

**Issue by kofemann** [dCache/dcache#5300]
On your test instance we see the following stacktrace
with current master (5366aca) 
```
30 Jan 2020 13:24:46 (Frontend-sisyphus) [] Uncaught exception in thread pool-55-thread-4
java.lang.NullPointerException: null
	at org.dcache.util.collector.pools.PoolInfoCollectorUtils.mergeLastAccess(PoolInfoCollectorUtils.java:326)
	at org.dcache.restful.util.pool.PoolHistoriesHandler.getAggregateFileLifetime(PoolHistoriesHandler.java:193)
	at org.dcache.restful.util.pool.PoolHistoriesHandler.update(PoolHistoriesHandler.java:265)
	at org.dcache.util.collector.pools.PoolInfoAggregator.lambda$aggregateDataForPoolGroups$0(PoolInfoAggregator.java:100)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.HashMap$ValueSpliterator.forEachRemaining(HashMap.java:1675)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
	at org.dcache.util.collector.pools.PoolInfoAggregator.aggregateDataForPoolGroups(PoolInfoAggregator.java:95)
	at org.dcache.restful.util.pool.PoolDataRequestProcessor.postProcess(PoolDataRequestProcessor.java:105)
	at org.dcache.util.collector.RequestFutureProcessor.waitUntilFinished(RequestFutureProcessor.java:233)
	at org.dcache.util.BoundedExecutor$Worker.run(BoundedExecutor.java:251)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by alrossi** [comment]
I thought

Patch: https://rb.dcache.org/r/12130/
Committed: master@3df1cd4302b738f71684cb977ff8e29a054f4a01

fixed this.   Damn.

**Comment by kofemann** [comment]
fixed with commit 3df1cd4302
